content/en/security/application_security/threats/security_signals.md

```diff
@@ -20,23 +20,96 @@ In the [Signals Explorer][2], filter by attributes and facets to find critical t
 
 {{< img src="security/application_security/threats/security_signals/appsec-threat-signals.png" alt="Overview of investigating threats in signals explorer with details side panel">}}
 
+## Signals Explorer columns
+
+The Signals Explorer displays signals using the following columns:
+
+### Severity
+
+There are five severity states: **Critical**, **High**, **Medium**, **Low**, and **Info**. 
+
+- **Critical:** A service is compromised. For example, a SQL injection. An immediate response is required.
```

Comment:
```suggestion
- **Critical:** A service is compromised. For example, an SQL injection. An immediate response is required.
```
Perhaps pedantic, but I think this is pronounced ES-QUEUE-EL (rather than "sequel"). Feel free to ignore. If there wasn't another small edit in the PR, I probably would have just approved.